contracts/allocs/mainnet.json

```diff
@@ -1347,7 +1347,7 @@
     }
   },
   "0x121e240000000000000000000000000000000002": {
```

Comment:
@kevinhalliday this is likely just the fact we sum two floats 0.01 + 0.01 != 0.02 so we maybe should change from floats

Reply:
yep that's what i did

Reply:
shouldn't the above value be updated then?